### DISCUSSION TITLE
how to use fisheye cam model  and distort model KB8

### DISCUSSION THREAD

**Discussion by lingjie12** [microsoft/AirSim#4239] [Support Q&A]
I want to use cam distortion model fisheye(KB8),but I find it only can describe radial-tangential with k1 k2 k3 p1 p2. So, how can we set parameters to change distort model as KB8??

**Comment by zimmy87** [comment]
Are you by chance using the plugin documented [here](https://docs.unrealengine.com/4.27/en-US/API/Plugins/OpenCVLensDistortion/)? If so, it looks like there's a Calibrator object that can generate lens distortion parameters. You can see an example of this object being used [here](https://gamedev.stackexchange.com/questions/196569/unreal-how-to-feed-images-into-open-cv-lens-distortion-calibrator), though it looks like the poster is having trouble with his code, so you may need to do some further experimentation to figure out how to properly use this object.
